ai_genomics/analysis/crunchbase_definitions.py

```diff
@@ -0,0 +1,129 @@
+# Script to test definitions in the crunchbase data
+
+import io
+import logging
+from typing import Set, Union
+
+import pandas as pd
+import numpy as np
+from toolz import pipe
+
+from ai_genomics.utils.crunchbase import fetch_crunchbase, parse_s3_table
+
+from ai_genomics import PROJECT_DIR, config
+
+
+def search_terms(abstract: str, terms: Set) -> Union[bool, float]:
+    """Checks if a string contains any of the terms in a set.
+
+    Args:
+        abstract: string to check
+        terms: set of terms to check for
+
+    Returns:
+        True if any of the terms are found, False otherwise (and
+        nan if we don't have an abstract)
+    """
+
+    if pd.isnull(abstract) is False:
+
+        return any(t in abstract.lower() for t in terms)
+
+    else:
+        return np.nan
+
+
+if __name__ == "__main__":
+
+    logging.info("Check organisations in relevant categories")
+
+    cb_cat_comps = pipe(fetch_crunchbase("org_cats"), parse_s3_table)
+
+    # Get set of organisations in each of the categories
+    gen_cats, ai_cats = [
+        set(cb_cat_comps.query(f"category_name=='{cat}'")["organization_id"])
+        for cat in ["genetics", "artificial intelligence"]
+    ]
+
+    logging.info(f"Genetics category organisations: {len(gen_cats)}")
+    logging.info(f"Artificial intelligence category organisations: {len(ai_cats)}")
+    logging.info(f"organisations in both categories:{len(gen_cats & ai_cats)}")
+
+    logging.info("Check organisations with relevant abstracts")
+    cb_comps = pipe(fetch_crunchbase("orgs"), parse_s3_table)
+
+    logging.info(f"total organisations: {len(cb_comps)/1e6} M")
+
+    ai_terms, genom_terms = [
+        config[category] for category in ["ai_cb_terms", "genom_cb_terms"]
+    ]
+
+    cb_comps["description_combined"] = [
+        str(descr_short) + "_" + str(descr_long)
+        for descr_short, descr_long in zip(
+            cb_comps["short_description"], cb_comps["long_description"]
+        )
+    ]
```

Comment:
Can simplify this with an f-string.
Also, do we need the `_` separator? could it cause an issue with the wordsearch?

```suggestion
    cb_comps["description_combined"] = [
        f"{descr_short} {descr_long}"
        for descr_short, descr_long in zip(
            cb_comps["short_description"], cb_comps["long_description"]
        )
    ]
```